ts/Dashboards/Plugins/DataGridSyncHandlers.ts

```diff
@@ -98,7 +98,7 @@ const configs: {
     handlers: {
         highlightHandler: [
             'highlightHandler',
-            function (this: DataGridComponent): void {
+            function (this: DataGridComponent): Function | void {
```

Comment:
Can we be more specific about the typing here? It looks like each `unregisterCursorListeners` looks similar